Source/ACE/Network/GameEvent/Events/GameEventViewContents.cs

```diff
@@ -1,13 +1,27 @@
-﻿namespace ACE.Network.GameEvent.Events
+﻿using ACE.Entity;
+using ACE.Entity.Enum;
+
+namespace ACE.Network.GameEvent.Events
 {
+    using System.Diagnostics;
+
     public class GameEventViewContents : GameEventMessage
     {
-        public GameEventViewContents(Session session, uint objectId)
+        public GameEventViewContents(Session session, AceObject container)
```

Comment:
feels like this should be a WorldObject.. AceObject is behind the scenes usually